modules/eks_nodegroup/locals.tf

```diff
@@ -24,6 +24,7 @@ locals {
       "lifecycle"     = var.is_spot_instance ? "spot" : "ondemand"
       "nodeGroupName" = local.get_nodegroup_postfix
       "environment"   = local.get_environment
+      "assign_pods_to_key_nodes" = var.assign_pods_to_key_nodes == true ? "allow_only_critical_pods" : "allow_any_pods"
```

Comment:
could I ask you to make fmt code before apply it?
terrafrom fmt in the folder

Reply:
Was applied fmt format.